source/tutorials/wordpress/web-server-install.rst

```diff
@@ -28,7 +28,7 @@ for more information.
 Install the web-server-basic bundle
```

Comment:
Change "web-server-basic" to "httpd" to match line 31 below?

Reply:
thank you!  that saves me a more public d'oh! moment